modules/welcome.php

```diff
@@ -24,7 +24,9 @@
  *  $Id$
  */
 
-@include(LIB_DIR.'/locale/' . Localisation::getCurrentUiLanguage() . '/fortunes.php');
+if (!ConfigHelper::checkValue(ConfigHelper::getConfig('phpui.hide_fortunes', false))) {
```

Comment:
Jeśli domyślnie jest _false_ to można:
```php
if (!ConfigHelper::checkConfig('phpui.hide_fortunes')) {
```
Poprawisz czy ja to zrobię?

Reply:
Poprawię jeszcze dzisiaj, po pracy.

Reply:
Poprawione